Documentation/ApiOverview/CachingFramework/Index.rst

```diff
@@ -4,25 +4,155 @@
+To avoid that, TYPO3 generates a so-called `cHash` parameter, which is a hash that basically signs the valid parameters
+for that request. So any parameter that validly influences the rendered page needs to be part of that `cHash`.
+
+With :ref:`Routing` you can configure TYPO3 not to display the `cHash` in your URLs in most cases.
+
+Various configuration options exist to configure the `cHash` behavior via :php:`$GLOBALS['TYPO3_CONF_VARS']['FE']['cacheHash']`
+in the file `LocalConfiguration.php` or `AdditionalConfiguration.php`:
+
+================================== ====================================================================================================================================================================
+Option                             Description
+================================== ====================================================================================================================================================================
+cachedParametersWhiteList          Only the given parameters will be evaluated in the cHash calculation. Example: tx_news_pi1[uid]
+requireCacheHashPresenceParameters Configure Parameters that require a cHash. If no cHash is given but one of the parameters are set, then TYPO3 triggers the configured cHash Error behavior
+excludedParameters                 The given parameters will be ignored in the cHash calculation. Example: L,tx_search_pi1[query]
+excludedParametersIfEmpty          Configure Parameters only being relevant for the cHash if there's an associated value available. Set excludeAllEmptyParameters to true to skip all empty parameters.
+excludeAllEmptyParameters          If true, all parameters which are relevant for cHash are only considered if they are non-empty.
+================================== ====================================================================================================================================================================
+
+All properties can be configured with an array of values. Besides exact matches (*equals*) it is possible to apply partial matches at
+the beginning of a parameter (*startsWith*) or inline occurrences (*contains*).
+
+URL parameter names are prefixed with the following indicators:
++ :php:`=` (*equals*): exact match, default behavior if not given
++ :php:`^` (*startsWith*): matching the beginning of a parameter name
++ :php:`~` (*contains*): matching any inline occurrence in a parameter name
+
+These indicators can be used for all previously existing sub-properties
+:php:`cachedParametersWhiteList`, :php:`excludedParameters`, :php:`excludedParametersIfEmpty`
+and :php:`requireCacheHashPresenceParameters`.
+
+Example (excerpt of `LocalConfiguration.php`)
+---------------------------------------------
+
+.. code-block:: php
+
+   $GLOBALS['TYPO3_CONF_VARS']['FE']['cacheHash'] = [
+     'excludedParameters' => [
+       'utm_source',
+       'utm_medium',
+       '^utm_', // making previous two obsolete
```

Comment:
I know where this came from but written like this it's a bit confusing. Also people tend to copy and paste so we should split this into two snippets and also mention that the 2nd version covers *all* `utm_` arguments, not only the two mentioned here.

Reply:
While reading the part below, I noticed that it's actually superfluous - as the examples directly below explain why and when `^` is a good idea. So I removed it here to avoid confusion.